ydb/docs/en/core/reference/ydb-cli/workload-transfer.md

```diff
@@ -76,9 +76,12 @@ Parameter name | Parameter Description | Default value
 `--consumer-threads`, `-t` | Number of consumer threads | `1`
 `--consumers`, `-c` | Number of consumers | `1`
 `--message-size`, `-m` | Message size in bytes. It is possible to specify in KB, MB, GB by adding suffixes `K`, `M`, `G` respectively | `10240`
-`--message-rate` | Target total write speed. In messages per second. Excludes the use of the `--byte-rate` | `0` parameter
-`--byte-rate` | Target total write speed. In bytes per second. Excludes the use of the --message-rate parameter. It is possible to specify in KB/s, MB/s, GB/s by adding suffixes `K`, `M`, `G` respectively | `0`
+`--message-rate` | Target total write speed. In messages per second. Excludes the use of the `--byte-rate` parameter | `0`
+`--byte-rate` | Target total write speed. In bytes per second. Excludes the use of the `--message-rate` parameter. It is possible to specify in KB/s, MB/s, GB/s by adding suffixes `K`, `M`, `G` respectively | `0`
 `--commit-period` | The period between `COMMIT` calls. In seconds | `10`
+`--commit-messages` | The period between `COMMIT` calls. In number of messages | `1000000`
+`--only-topic-in-tx` | Only topic partitions are forced to participate in transactions Excludes the use of the `--only-table-in-tx` parameter | `0`
```

Comment:
A point symbol is missed befor "Excludes".
"in transactions <missed point> Excludes"

Reply:
> A point symbol is missed befor "Excludes". "in transactions Excludes"
✅